crunchy.go

```diff
@@ -186,6 +193,20 @@ func (v *Validator) Check(password string) error {
 		}
 	}
 
+	if v.options.MustContainDigit {
```

Comment:
As these two checks are relatively inexpensive we should probably move them ahead of some of the other checks. Probably between the check for `ErrTooFewChars` and `ErrTooSystematic`.

Reply:
@muesli thank you, happy to contribute. I moved those two checks in between the check for `ErrTooFewChars` and `ErrTooSystematic` as you suggested.